releases/v0.1.yml

```diff
@@ -0,0 +1,109 @@
+# This YAML file contains driver-registrar & csi driver nodeplugin API objects,
+# which are necessary to run csi nodeplugin for bizflycloud.
+
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-bizflycloud-nodeplugin
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: csi-bizflycloud-nodeplugin
+  template:
+    metadata:
+      labels:
+        app: csi-bizflycloud-nodeplugin
+    spec:
+      serviceAccount: csi-bizflycloud-node-sa
+      hostNetwork: true
+      containers:
+        - name: node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/volume.csi.bizflycloud.vn /registration/volume.csi.bizflycloud.vn-reg.sock"]
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/volume.csi.bizflycloud.vn/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+            - name: registration-dir
+              mountPath: /registration
+        - name: bizflycloud-csi-plugin
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: cr-hn-1.vccloud.vn/31ff9581861a4d0ea4df5e7dda0f665d/csi-bizflycloud:latest
```

Comment:
Why do you use `latest` tag for `v0.1` version? Can you make a new directory for Controller and Node Plugin manifest? I think we should move all manifests in manifest folder example to that folder.